osu.Game/Overlays/Comments/CommentsHeader.cs

```diff
@@ -11,6 +11,8 @@
 using osuTK;
 using osu.Framework.Input.Events;
 using osu.Game.Graphics.Sprites;
+using System.ComponentModel;
+using Container = osu.Framework.Graphics.Containers.Container;
```

Comment:
It's a nitpick but I'd prefer this the other way round:
```csharp
using osu.Framework.Graphics.Containers;
using Description = System.ComponentModel.Description;
```
or even use the fully qualified name.